.github/workflows/release.yml

```
@@ -27,9 +27,30 @@ jobs:
       - name: Check
         run: twine check dist/*
```

Note that the pypi-publish runs
```suggestion
        run: twine check --strict dist/*
```
and if your non-strict check will succeed, it may fail the upload late in the process which tends to be annoying. So I always recommend doing a strict check during normal linting. This allows catching problems and addressing them early.